sympy/geometry/polygon.py

```diff
@@ -438,6 +438,70 @@ def second_moment_of_area(self, point=None):
         return I_xx, I_yy, I_xy
 
 
+    def polar_modulus(self):
+        """Returns the polar modulus of a two-dimensional polygon
+
+        Examples
+        ========
+
+        >>> from sympy import Polygon, symbols
+        >>> a, b = symbols('a, b')
+        >>> rectangle = Polygon((0, 0), (a, 0), (a, b), (0, b))
+        >>> rectangle.polar_modulus()
+        a**3*b/12 + a*b**3/12
+        """
+        return self.second_moment_of_area()[0] + self.second_moment_of_area()[1]
```

Comment:
Can we do this by only calling second_moment_of_area once?

Reply:
yes, I will be changing it